src/python/WMQuality/Emulators/SiteDBClient/MakeSiteDBMockFiles.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+"""
+MakeSiteDBMockFiles
+Program to create mock SiteDB JSON files used by the SiteDB mock-based emulator
+"""
+
+from __future__ import (division, print_function)
+from WMCore.Services.SiteDB.SiteDBAPI import SiteDBAPI
+from WMCore.WMBase import getTestBase
+from urllib2 import HTTPError
+import json
+import os
+
+if __name__ == '__main__':
+    calls = [{'callname': 'people', 'filename': 'people.json', 'clearCache': False, 'verb': 'GET', 'data':{}},
+             {'callname': 'site-names', 'filename': 'site-names.json', 'clearCache': False, 'verb': 'GET', 'data':{}},
+             {'callname': 'site-resources', 'filename': 'site-resources.json', 'clearCache': False, 'verb': 'GET', 'data':{}},
+             {'callname': 'data-processing', 'filename': 'data-processing.json', 'clearCache': False, 'verb': 'GET', 'data':{}}
+            ]
+
+    dns = ["/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=liviof/CN=472739/CN=Livio Fano'",
+           "/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=jha/CN=618566/CN=Manoj Jha"]
+    lookup = {}
+
+    outFile = 'SiteDBMockData.json'
+    outFilename = os.path.join(getTestBase(), '..', 'data', 'Mock', outFile)
+    print(outFilename)
+
+    x = SiteDBAPI()
+    for call in calls:
+        signature = str(sorted(call.iteritems()))
+        try:
+            result = x.getJSON(**call)
+        except HTTPError:
+            result = 'Raises HTTPError'
+
+        if call['callname'] == 'people':
+            tmp = []
```

Comment:
I think there is a more elegant way to do this with list and/or dictionary comprehensions. Check that out